src/QuickMarcEditor/QuickMarcEditorRows/LinkButton/LinkButton.test.js

```diff
@@ -347,7 +347,7 @@ describe('Given LinkButton', () => {
       });
 
       expect(Pluggable).toHaveBeenLastCalledWith(expect.objectContaining({
-        tenantId: '',
+        tenantId: null,

```

Comment:
I think it is a `useOkapiKy` issue, it shouldn't allow the `tenantId` to be `''`, `0` and `false`.